### PR TITLE
Orion Solang

### PR DESCRIPTION
**Pull request by orionslg**

### CODE REVIEW

**Comment by orionslg** [comment]
https://kelontongmart.firebaseapp.com/

email: admin@gmail.com
password: 123456

File: src/components/dashboard/form/AddProduct.vue [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+        // Optional parameters
+        container: null,
+        canCancel: true,
+        onCancel: this.onCancel,
+      });
+      this.$store.dispatch('addProduct', this.productData)
+        .then(() => {
+          loader.hide();
+
+          this.$toasted.show('New category added', {
```
Comment (ziadyjo) [inline src/components/dashboard/form/AddProduct.vue:137]: New product added ya, mungkin lupa keganti

File: src/views/Dashboard.vue [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            </li>
+          </ul>
+        </div>
+      </nav>
+      <div class="d-flex flex-column" id="content-wrapper">
+        <div id="content">
+          <nav class="navbar navbar-light navbar-expand bg-white shadow mb-4 topbar static-top">
+            <div class="container-fluid"><button class="btn btn-link d-md-none rounded-circle mr-3"
+                id="sidebarToggleTop" type="button"><i class="fas fa-bars"></i></button>
+              <form
```
Comment (ziadyjo) [inline src/views/Dashboard.vue:42]: Ini memang belum berfungsi ya

File: src/components/dashboard/table/product.vue [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+                <option value="100">100</option>
+              </select>&nbsp;</label></div>
+        </div>
+        <div class="col-md-6">
+          <div class="text-md-right dataTables_filter" id="dataTable_filter"><label><input
+                type="search" class="form-control form-control-sm" aria-controls="dataTable"
+                placeholder="Search"></label></div>
+        </div>
+      </div>
+      <div class="table-responsive table mt-2" id="dataTable" role="grid"
```
Comment (ziadyjo) [inline src/components/dashboard/table/product.vue:25]: nice pake data table dan pagination

**Pull request closed without merging**